ToDoList/src/model/Status.java

```diff
@@ -0,0 +1,13 @@
+package model;
+
+/**
+ * The status of an <code>Element</code>
+ * @author Krishna Sannasi
+ *
+ */
+public enum Status

```

Comment:
Note that if the option is "in progress" we need to include a date started, also if the option is "finished" we need to include a date of completion.

Reply:
Yeah,  I'll see how to work that in.